tests/preserve_sequence.rs

```diff
@@ -0,0 +1,59 @@
+use ron::ser::{to_string_pretty, PrettyConfig};
+use ron::de::from_reader;
+use serde::Deserialize;
+use serde::Serialize;
+use std::{collections::HashMap, fs::File};
+use std::env;
+use std::fs;
+
+#[derive(Debug, Deserialize, Serialize)]
+struct Config {
+    boolean: bool,
+    float: f32,
+    map: HashMap<u8, char>,
+    nested: Nested,
+    tuple: (u32, u32),
+}
+
+#[derive(Debug, Deserialize, Serialize)]
+struct Nested {
+    a: String,
+    b: char,
+}
+
+fn read_original(file_name: &str) -> (String) {
+    let input_path = format!("{}/tests/{}", env!("CARGO_MANIFEST_DIR"), file_name);
+    fs::read_to_string(input_path)
+        .expect("Something went wrong reading the file")
+}
+
+fn test_sequence(file_name: &str) -> (String) {
```

Comment:
perhaps this should be called `test_roundtrip` instead?

Reply:
Good suggestion. I though about several names (reciprocal, symmetric etc) but couldn't find a brief and correct description. See 1272b61ce01ae321d3e0050bdf6b6d87b8ea61d1

Reply:
Forgot to update calling functions, see update in e3ada7f0c2876a598b17888cab2f9e445b3c9632 Now the first example fails as the mapping order is modified, while the second example proofs the test is okay.